homeassistant/components/sensor/loxone.py

```diff
@@ -0,0 +1,89 @@
+"""
+Loxone simple sensor
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.loxone/
+"""
+
+import logging
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (CONF_UNIT_OF_MEASUREMENT)
+
+DOMAIN = 'loxone'
+EVENT = 'loxone_received'
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_SENSOR_NAME = 'sensorname'
+CONF_SENSOR_TYP = 'sensortyp'
+CONF_UUID = 'uuid'
+
+CONFIG_SCHEMA = vol.Schema({
```

Comment:
You have to extend the sensor validation. Get the details from the link I provided in the previous comment about the configuration validation.